src/background/stores/tab-store.ts

```diff
@@ -85,11 +91,11 @@ export class TabStore extends BaseStoreImpl<TabStoreData> {
         }
     };
 
-    private originsMatch = (url1: string, url2: string): boolean => {
-        if (url1 == null && url2 == null) {
+    private originsMatch = (url1: string | null, url2: string | null): boolean => {
+        if (url1 === null && url2 === null) {
```

Comment:
In most cases we do prefer to use `===` over `==`, but comparisons against `null` are usually intentional exceptions; when you're comparing against `null` specifically, the difference is that `==` will cause `undefined` to be treated the same as `null` where `===` will cause it to be treated differently. It's very rare that we want to treat those two differently in practice.